### PR TITLE
remove techpreview upgrade ci jobs

### PR DESCRIPTION
**Pull request by jianlinliu**
techpreview is not supported between major versions upgrade, so drop them.

### CODE REVIEW

**Comment by jianlinliu** [comment]
cc @liangxia for review.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml [hunk 1]
```diff
@@ -328,18 +328,6 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-private
-- as: azure-ipi-proxy-techpreview-p2-f14
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml:331]: Looks like you forgot to run `make update` ?

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml:331]: fixed.

**Pull request merged**